### ISSUE TITLE
Aura ring not rotating with token

### ISSUE THREAD

**Issue by Orionox** [AnthonyEdmonds/token-aura-ring#16]
When I turn my token to the right, the aura jumps to a new angle and no longer rotates normally with the token. When I rotate the token to the left it acts as expected.

**Comment by Deyzeria** [comment]
Yeah, directional aura ring is not rotating correctly with the token. In my case even just rotating using shift+mouse wheel makes the directional aura ring jump in all sorts of places but almost never correctly, almost as if randomly.

**Comment by AnthonyEdmonds** [comment]
I have almost finished with my UI overhaul; looking at it, the aura rotation seems to be one update behind for some reason.

**Comment by Orionox** [comment]
For me it only happens when rotating in a single direction

**Comment by AnthonyEdmonds** [comment]
This should be resolved with v2.0.0. If it isn't, you can re-open the ticket.
